oci/private/push.sh.tpl

```diff
@@ -52,13 +52,13 @@ done
 DIGEST=$("${JQ}" -r '.manifests[0].digest' "${IMAGE_DIR}/index.json")
 
 REFS=$(mktemp)
-"${CRANE}" push ${VERBOSE} "${IMAGE_DIR}" "${REPOSITORY}@${DIGEST}" "${ARGS[@]+"${ARGS[@]}"}" --image-refs "${REFS}"
```

Comment:
This change was not strictly necessary, but since we have `ARGS=()` above, the `+` check seems redundant.

Reply:
What exactly would happen with an empty array with `"${ARGS[@]}"`? Wouldn't it just evaluate to the empty string? But I can take it back for sure.

Reply:
depends on flavor of bash, old bash implementations complain, i don't remember exact error message.